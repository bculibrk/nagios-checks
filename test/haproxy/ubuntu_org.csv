# pxname,svname,qcur,qmax,scur,smax,slim,stot,bin,bout,dreq,dresp,ereq,econ,eresp,wretr,wredis,status,weight,act,bck,chkfail,chkdown,lastchg,downtime,qlimit,pid,iid,sid,throttle,lbtot,tracked,type,rate,rate_lim,rate_max,check_status,check_code,check_duration,hrsp_1xx,hrsp_2xx,hrsp_3xx,hrsp_4xx,hrsp_5xx,hrsp_other,hanafail,req_rate,req_rate_max,req_tot,cli_abrt,srv_abrt,comp_in,comp_out,comp_byp,comp_rsp,lastsess,last_chk,last_agt,qtime,ctime,rtime,ttime,
https-in,FRONTEND,,,2,85,4096,2136332,2504775167,82899157497,0,0,98854,,,,,OPEN,,,,,,,,,1,2,0,,,,0,0,0,34,,,,0,6231043,49724,158930,4502,3758,,1,93,6447957,,,0,0,0,0,,,,,,,,
https-in,ipv4,,,2,85,4096,2169569,2430958277,78640483024,0,0,94649,,,,,OPEN,,,,,,,,,1,2,1,,,,3,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
https-in,ipv6,,,0,23,4096,43891,73816890,4258674473,0,0,4205,,,,,OPEN,,,,,,,,,1,2,2,,,,3,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
http-in,FRONTEND,,,1,109,4096,940259,549849173,7477022674,0,0,11356,,,,,OPEN,,,,,,,,,1,3,0,,,,0,1,0,82,,,,0,615286,454818,103254,471,247,,1,82,1174077,,,0,0,0,0,,,,,,,,
http-in,ipv4,,,1,109,4096,918905,532578634,7316902232,0,0,8617,,,,,OPEN,,,,,,,,,1,3,1,,,,3,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
http-in,ipv6,,,0,28,4096,21354,17270539,160120442,0,0,2739,,,,,OPEN,,,,,,,,,1,3,2,,,,3,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
ubuntu-de,verein,0,0,0,4,256,18029,4026100,8989659,,0,,0,0,0,0,UP,1,1,0,0,0,2885976,0,,1,4,1,,18029,,2,0,,13,L4OK,,1,0,0,18029,0,0,0,0,,,,0,0,,,,,53,,,0,0,1,166,
ubuntu-de,BACKEND,0,0,0,4,820,18029,4026100,8989659,0,0,,0,0,0,0,UP,1,1,0,,0,2885976,0,,1,4,0,,18029,,1,0,,13,,,,0,0,18029,0,0,0,,,,,0,0,0,0,0,0,53,,,0,0,1,166,
verein-ubuntu-de,verein,0,0,0,5,256,17009,3522633,121239768,,0,,0,0,0,0,UP,1,1,0,0,0,2885976,0,,1,5,1,,17009,,2,0,,12,L4OK,,0,0,14769,98,2109,0,0,0,,,,18,0,,,,,236,,,0,0,2,320,
verein-ubuntu-de,BACKEND,0,0,0,5,820,17009,3522633,121239768,0,0,,0,0,0,0,UP,1,1,0,,0,2885976,0,,1,5,0,,17009,,1,0,,12,,,,0,14769,98,2123,4,15,,,,,18,0,0,0,0,0,236,,,0,0,2,320,
ubuconde,verein,0,0,0,8,256,121764,38989902,1655707530,,0,,0,0,0,0,UP,1,1,0,0,0,2885976,0,,1,6,1,,121764,,2,0,,43,L4OK,,0,0,73056,22758,25922,0,0,0,,,,79,0,,,,,111,,,0,0,15,467,
ubuconde,BACKEND,0,0,0,8,820,220280,69206218,1670552828,0,0,,0,0,0,0,UP,1,1,0,,0,2885976,0,,1,6,0,,121764,,1,0,,43,,,,0,73056,121274,25930,2,18,,,,,79,0,0,0,0,0,111,,,0,0,15,467,
ubuconeu,verein,0,0,0,0,256,0,0,0,,0,,0,0,0,0,UP,1,1,0,0,0,2885976,0,,1,7,1,,0,,2,0,,0,L4OK,,0,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuconeu,BACKEND,0,0,0,1,820,4,574,481,0,0,,0,0,0,0,UP,1,1,0,,0,2885976,0,,1,7,0,,0,,1,0,,1,,,,0,0,4,0,0,0,,,,,0,0,0,0,0,0,-1,,,0,0,0,0,
ubuntu-berlin,berlin,0,0,0,8,256,33023,7862551,92403133,,0,,0,0,0,0,UP,1,1,0,1,0,2885976,0,,1,8,1,,33023,,2,0,,31,L4OK,,0,0,11187,104,21722,0,0,0,,,,7,0,,,,,53,,,0,1,2,115,
ubuntu-berlin,BACKEND,0,0,0,8,820,33023,7862551,92403133,0,0,,0,0,0,0,UP,1,1,0,,0,2885976,0,,1,8,0,,33023,,1,0,,31,,,,0,11187,104,21729,3,0,,,,,7,0,0,0,0,0,53,,,0,1,2,115,
ubuntuusers-staging,webteam,0,0,0,8,128,7757,4285547,42440176,,0,,0,4,0,0,UP,1,1,0,0,0,2885976,0,,1,9,1,,7757,,2,0,,41,L4OK,,1,0,4658,2206,799,0,0,0,,,,21,0,,,,,21953,,,0,0,155,511,
ubuntuusers-staging,BACKEND,0,0,0,8,820,7757,4285547,42440176,0,0,,0,4,0,0,UP,1,1,0,,0,2885976,0,,1,9,0,,7757,,1,0,,41,,,,0,4658,2206,820,32,41,,,,,21,0,0,0,0,0,21953,,,0,0,155,511,
ubuntuusers-testing,webteam,0,0,0,2,128,544,147021,4884452,,0,,0,0,0,0,UP,1,1,0,0,0,2885976,0,,1,10,1,,544,,2,0,,8,L4OK,,1,0,108,25,411,0,0,0,,,,0,0,,,,,844,,,0,0,154,247,
ubuntuusers-testing,BACKEND,0,0,0,2,820,544,147021,4884452,0,0,,0,0,0,0,UP,1,1,0,,0,2885976,0,,1,10,0,,544,,1,0,,8,,,,0,108,25,411,0,0,,,,,0,0,0,0,0,0,844,,,0,0,154,247,
ubuntuusers-oss,webteam,0,0,0,2,128,586,152457,4763156,,0,,0,0,0,0,UP,1,1,0,0,0,2885976,0,,1,11,1,,586,,2,0,,8,L4OK,,1,0,126,45,414,0,0,0,,,,2,0,,,,,19076,,,0,1,195,299,
ubuntuusers-oss,BACKEND,0,0,0,2,820,586,152457,4763156,0,0,,0,0,0,0,UP,1,1,0,,0,2885976,0,,1,11,0,,586,,1,0,,8,,,,0,126,45,415,0,0,,,,,2,0,0,0,0,0,19076,,,0,1,195,299,
ubuntuusers-tt,srv10,0,0,0,0,512,0,0,0,,0,,0,0,0,0,DOWN,1,1,0,1,1,2885975,2885975,,1,12,1,,0,,2,0,,0,L4CON,,0,0,0,0,0,0,0,0,,,,0,0,,,,,-1,Connection refused,,0,0,0,0,
ubuntuusers-tt,BACKEND,0,0,0,1,820,27,4389,369603,0,0,,27,0,0,0,DOWN,0,0,0,,1,2885975,2885975,,1,12,0,,0,,1,0,,1,,,,0,0,0,0,27,0,,,,,0,0,0,0,0,0,-1,,,0,0,0,0,
kubuntu-de,kubuntude,0,0,0,102,256,643276,409335508,5027529706,,0,,0,75,1,0,UP,1,1,0,1,0,2885976,0,,1,13,1,,643275,,2,0,,82,L4OK,,0,0,570189,5375,43042,1,0,0,,,,25399,3,,,,,17,,,0,1,1337,1895,
kubuntu-de,BACKEND,0,0,0,102,410,643276,409335997,5027529706,0,0,,0,75,1,0,UP,1,1,0,,0,2885976,0,,1,13,0,,643275,,1,0,,82,,,,0,570189,5375,67289,217,206,,,,,25400,3,0,0,0,0,17,,,0,1,1337,1895,
ubuntuir,ubuntuir,0,0,0,31,256,6173131,2420515183,77345298456,,0,,0,2865,2,0,UP,1,1,0,3,0,2885976,0,,1,14,1,,6173129,,2,1,,93,L4OK,,0,0,6113357,23903,18996,36,0,0,,,,40835,3,,,,,1,,,0,0,273,976,
ubuntuir,BACKEND,0,0,0,32,820,6466201,2500450914,77390583352,0,0,,0,2865,2,0,UP,1,1,0,,0,2885976,0,,1,14,0,,6173129,,1,1,,93,,,,0,6113357,316975,27844,4457,3568,,,,,40835,3,0,0,0,0,1,,,0,0,273,976,
mailman,mailman,0,0,0,3,256,20632,5688058,34401877,,0,,0,0,0,0,UP,1,1,0,0,0,2885976,0,,1,15,1,,20632,,2,0,,7,L4OK,,1,0,17450,397,2780,0,0,0,,,,8,0,,,,,249,,,0,1,8,1933,
mailman,BACKEND,0,0,0,3,820,45031,9406698,37587101,0,0,,0,0,0,0,UP,1,1,0,,0,2885976,0,,1,15,0,,20632,,1,0,,9,,,,0,17450,24796,2783,0,2,,,,,8,0,0,0,0,0,249,,,0,1,8,1933,
jenkins,webteam,0,0,0,10,128,17483,33459279,60175774,,0,,0,0,0,0,UP,1,1,0,5,1,71039,30,,1,16,1,,17483,,2,0,,78,L4OK,,1,0,15363,1499,287,14,0,0,,,,1123,0,,,,,1367,,,0,0,32,566,
jenkins,BACKEND,0,0,0,10,820,17505,33464331,60178605,0,0,,0,0,0,0,UP,1,1,0,,1,71039,30,,1,16,0,,17483,,1,0,,78,,,,0,15363,1521,459,19,143,,,,,1123,0,0,0,0,0,1367,,,0,0,32,566,
sentry,sentry,0,0,0,9,128,1149,1334051,26779983,,0,,0,0,0,0,UP,1,1,0,0,0,2885976,0,,1,17,1,,1149,,2,0,,27,L4OK,,0,0,980,73,66,0,0,0,,,,18,0,,,,,99568,,,0,0,134,699,
sentry,BACKEND,0,0,0,9,820,1158,1336010,26781170,0,0,,0,0,0,0,UP,1,1,0,,0,2885976,0,,1,17,0,,1149,,1,0,,27,,,,0,980,82,84,0,12,,,,,18,0,0,0,0,0,99568,,,0,0,134,699,
znc,f01,0,0,0,0,128,0,0,0,,0,,0,0,0,0,UP,1,1,0,0,0,2885976,0,,1,18,1,,0,,2,0,,0,L4OK,,0,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
znc,BACKEND,0,0,0,0,820,0,0,0,0,0,,0,0,0,0,UP,1,1,0,,0,2885976,0,,1,18,0,,0,,1,0,,0,,,,0,0,0,0,0,0,,,,,0,0,0,0,0,0,-1,,,0,0,0,0,
ubuntuusers-python,ruwa,0,0,0,0,256,0,0,0,,0,,0,0,0,0,UP,50,1,0,13,3,2279139,797,,1,19,1,,0,,2,0,,0,L4OK,,7,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-python,tali,0,0,0,0,256,0,0,0,,0,,0,0,0,0,UP,50,1,0,17,3,2279061,741,,1,19,2,,0,,2,0,,0,L4OK,,7,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-python,BACKEND,0,0,0,2,820,5067,1670671,1137687,0,0,,0,0,0,0,UP,100,2,0,,3,2279139,627,,1,19,0,,0,,1,0,,10,,,,0,0,5067,0,0,0,,,,,0,0,0,0,0,0,-1,,,0,0,0,0,
ubuntuusers-static,she,0,0,0,0,512,0,0,0,,0,,0,0,0,0,UP,1,1,0,46,8,761228,267,,1,20,1,,0,,2,0,,0,L4OK,,7,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-static,kanu,0,0,0,0,512,0,0,0,,0,,0,0,0,0,DOWN,1,1,0,1,1,2885972,2885972,,1,20,2,,0,,2,0,,0,L4TOUT,,2001,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-static,BACKEND,0,0,0,2,820,1686,396922,239897,0,0,,0,0,0,0,UP,1,1,0,,8,761228,267,,1,20,0,,0,,1,0,,3,,,,0,0,1686,0,0,0,,,,,0,0,0,0,0,0,-1,,,0,0,0,0,
ubuntuusers-staticwiki,verein,0,0,0,6,256,5388,1851889,3367791133,,0,,0,0,0,0,UP,1,1,0,1,0,2885976,0,,1,21,1,,5388,,2,0,,66,L4OK,,1,0,3289,12,2085,0,0,0,,,,12,0,,,,,3762,,,0,1,4,1137,
ubuntuusers-staticwiki,BACKEND,0,0,0,6,820,6923,2297262,3368009839,0,0,,0,0,0,0,UP,1,1,0,,0,2885976,0,,1,21,0,,5388,,1,0,,66,,,,0,3289,1547,2087,0,0,,,,,12,0,0,0,0,0,3762,,,0,1,4,1137,
ubuntuusers-media,she,0,0,0,0,512,0,0,0,,0,,0,0,0,0,UP,1,1,0,50,9,761227,273,,1,22,1,,0,,2,0,,0,L4OK,,7,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-media,kanu,0,0,0,0,512,0,0,0,,0,,0,0,0,0,DOWN,1,1,0,1,1,2885972,2885972,,1,22,2,,0,,2,0,,0,L4TOUT,,2001,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-media,BACKEND,0,0,0,0,820,0,0,0,0,0,,0,0,0,0,UP,1,1,0,,9,761227,273,,1,22,0,,0,,1,0,,0,,,,0,0,0,0,0,0,,,,,0,0,0,0,0,0,-1,,,0,0,0,0,
ubuntuusers-tour,she,0,0,0,0,512,0,0,0,,0,,0,0,0,0,UP,1,1,0,39,7,761228,291,,1,23,1,,0,,2,0,,0,L4OK,,7,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-tour,kanu,0,0,0,0,512,0,0,0,,0,,0,0,0,0,DOWN,1,1,0,1,1,2885972,2885972,,1,23,2,,0,,2,0,,0,L4TOUT,,2001,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-tour,BACKEND,0,0,0,0,820,0,0,0,0,0,,0,0,0,0,UP,1,1,0,,7,761228,291,,1,23,0,,0,,1,0,,0,,,,0,0,0,0,0,0,,,,,0,0,0,0,0,0,-1,,,0,0,0,0,
ubuntuusers-legacy,she,0,0,0,0,512,0,0,0,,0,,0,0,0,0,UP,1,1,0,45,10,761228,276,,1,24,1,,0,,2,0,,0,L4OK,,7,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-legacy,kanu,0,0,0,0,512,0,0,0,,0,,0,0,0,0,DOWN,1,1,0,1,1,2885972,2885972,,1,24,2,,0,,2,0,,0,L4TOUT,,2001,0,0,0,0,0,0,0,,,,0,0,,,,,-1,,,0,0,0,0,
ubuntuusers-legacy,BACKEND,0,0,0,0,820,0,0,0,0,0,,0,0,0,0,UP,1,1,0,,10,761228,276,,1,24,0,,0,,1,0,,0,,,,0,0,0,0,0,0,,,,,0,0,0,0,0,0,-1,,,0,0,0,0,
stats,BACKEND,0,0,1,19,820,27604,6754975,2497461784,0,0,,212,0,0,0,UP,0,0,0,,0,2885976,0,,1,26,0,,0,,1,1,,34,,,,0,21683,5708,0,212,0,,,,,45,0,0,0,0,0,0,,,1373,0,0,579,
