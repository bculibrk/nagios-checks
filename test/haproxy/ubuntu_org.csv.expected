HAPROXY CRIT: ubuntuusers-tt BACKEND (act) DOWN	sess=0/820(0%) smax=1
http-in FRONTEND (act) OPEN	sess=1/4096(0%) smax=109
http-in ipv4 (act) OPEN	sess=1/4096(0%) smax=109
http-in ipv6 (act) OPEN	sess=0/4096(0%) smax=28
https-in FRONTEND (act) OPEN	sess=2/4096(0%) smax=85
https-in ipv4 (act) OPEN	sess=2/4096(0%) smax=85
https-in ipv6 (act) OPEN	sess=0/4096(0%) smax=23
jenkins BACKEND (act) UP	sess=0/820(0%) smax=10
jenkins webteam (act) UP	sess=0/128(0%) smax=10
kubuntu-de BACKEND (act) UP	sess=0/410(0%) smax=102
kubuntu-de kubuntude (act) UP	sess=0/256(0%) smax=102
mailman BACKEND (act) UP	sess=0/820(0%) smax=3
mailman mailman (act) UP	sess=0/256(0%) smax=3
sentry BACKEND (act) UP	sess=0/820(0%) smax=9
sentry sentry (act) UP	sess=0/128(0%) smax=9
ubuconde BACKEND (act) UP	sess=0/820(0%) smax=8
ubuconde verein (act) UP	sess=0/256(0%) smax=8
ubuconeu BACKEND (act) UP	sess=0/820(0%) smax=1
ubuconeu verein (act) UP	sess=0/256(0%) smax=0
ubuntu-berlin BACKEND (act) UP	sess=0/820(0%) smax=8
ubuntu-berlin berlin (act) UP	sess=0/256(0%) smax=8
ubuntu-de BACKEND (act) UP	sess=0/820(0%) smax=4
ubuntu-de verein (act) UP	sess=0/256(0%) smax=4
ubuntuir BACKEND (act) UP	sess=0/820(0%) smax=32
ubuntuir ubuntuir (act) UP	sess=0/256(0%) smax=31
ubuntuusers-legacy BACKEND (act) UP	sess=0/820(0%) smax=0
ubuntuusers-legacy kanu (act) DOWN	sess=0/512(0%) smax=0
ubuntuusers-legacy she (act) UP	sess=0/512(0%) smax=0
ubuntuusers-media BACKEND (act) UP	sess=0/820(0%) smax=0
ubuntuusers-media kanu (act) DOWN	sess=0/512(0%) smax=0
ubuntuusers-media she (act) UP	sess=0/512(0%) smax=0
ubuntuusers-oss BACKEND (act) UP	sess=0/820(0%) smax=2
ubuntuusers-oss webteam (act) UP	sess=0/128(0%) smax=2
ubuntuusers-python BACKEND (act) UP	sess=0/820(0%) smax=2
ubuntuusers-python ruwa (act) UP	sess=0/256(0%) smax=0
ubuntuusers-python tali (act) UP	sess=0/256(0%) smax=0
ubuntuusers-staging BACKEND (act) UP	sess=0/820(0%) smax=8
ubuntuusers-staging webteam (act) UP	sess=0/128(0%) smax=8
ubuntuusers-static BACKEND (act) UP	sess=0/820(0%) smax=2
ubuntuusers-static kanu (act) DOWN	sess=0/512(0%) smax=0
ubuntuusers-static she (act) UP	sess=0/512(0%) smax=0
ubuntuusers-staticwiki BACKEND (act) UP	sess=0/820(0%) smax=6
ubuntuusers-staticwiki verein (act) UP	sess=0/256(0%) smax=6
ubuntuusers-testing BACKEND (act) UP	sess=0/820(0%) smax=2
ubuntuusers-testing webteam (act) UP	sess=0/128(0%) smax=2
ubuntuusers-tour BACKEND (act) UP	sess=0/820(0%) smax=0
ubuntuusers-tour kanu (act) DOWN	sess=0/512(0%) smax=0
ubuntuusers-tour she (act) UP	sess=0/512(0%) smax=0
ubuntuusers-tt BACKEND (act) DOWN	sess=0/820(0%) smax=1
ubuntuusers-tt srv10 (act) DOWN	sess=0/512(0%) smax=0
verein-ubuntu-de BACKEND (act) UP	sess=0/820(0%) smax=5
verein-ubuntu-de verein (act) UP	sess=0/256(0%) smax=5
znc BACKEND (act) UP	sess=0/820(0%) smax=0
znc f01 (act) UP	sess=0/128(0%) smax=0
